compiler-plugin/src/main/scala/scalapb/compiler/ProtobufGenerator.scala

```diff
@@ -23,6 +23,22 @@ class ProtobufGenerator(
   import DescriptorImplicits.AsSymbolExtension
   import ProtobufGenerator._
 
+  private val CompSeqType = {
```

Comment:
I prefer that constants will go inside `object C {}` (I know it's currently inconsistent). Since this constant depends on the parameteres, can you change `object C` to `class Constants(params: ...)`, and inside `class ProtobufGenerator` have `private val C = new Constants(params)` so the change is minimized?